spec/lita/adapters/slack/api_spec.rb

```diff
@@ -818,25 +818,25 @@ def stubs(postMessage_options = {})
       end
 
       it "has data on the bot user" do
-        response = subject.rtm_start
+        response = subject.rtm_connect
 
         expect(response.self.id).to eq('U12345678')
       end
 
       it "has an array of IMs" do
-        response = subject.rtm_start
+        response = subject.rtm_connect
```

Comment:
@lizcorgi Good call. In that case, we can probably get rid of this test and the one below as well, right?

Reply:
I think we can change the mocked response to match the proper response we get now and then, yes some of those tests would be invalid since a lot of these elements aren't in the new response.

Reply:
I created issue #5 to track this cleanup.